<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundOptions</class>
 <widget class="QDialog" name="BackgroundOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Background Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Calibration Method</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbLinear">
          <property name="text">
           <string>Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbRational">
          <property name="text">
           <string>Rational</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="laCalibration">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>RGB Background Calibration Method</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>Minimum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string>Middle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_6">
          <property name="text">
           <string>Maximum</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="laRGBCalibration">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLinear</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>onToggledLinear(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRational</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>onToggledRational(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>onToggledNone(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>onToggledMinimum(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>onToggledMiddle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundOptions</receiver>
   <slot>onToggledMaximum(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onToggledLinear(bool)</slot>
  <slot>onToggledRational(bool)</slot>
  <slot>onToggledNone(bool)</slot>
  <slot>onToggledMinimum(bool)</slot>
  <slot>onToggledMedium(bool)</slot>
  <slot>onToggledMaximum(bool)</slot>
 </slots>
</ui>
